src/components/summary/_macro.njk

```diff
@@ -9,26 +11,27 @@
     {% if params.hub %}
         {% set className = className + " ons-summary--hub" %}
     {% endif %}
+
     <div class="{{ className }}">
         {% for summary in params.summaries %}
             {% if summary.summaryTitle %}
                 <h2 class="ons-summary__title ons-u-mb-m{{ " ons-u-mt-m" if loop.index > 1 else "" }}">{{ summary.summaryTitle }}</h2>
                 {% set titleSize = "3" %}
             {% endif %}
             {% for group in summary.groups %}
-                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="ons-summary__group">
+                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="ons-summary__group {{"ons-summary__group--card" if cardVariant}}">
```

Comment:
```suggestion
                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="ons-summary__group{{ variant }}">
```
Along with this

Reply:
Can we discuss this call tomorrow. I have few doubts regarding this